### PR TITLE
Update peerDependencies

### LINKED ISSUE #309: Deprecation warning when installing react-konva in a clean project
After installing `react-konva` in a clean project (with `konva`, `react` & `react-dom` installed), I get the following deprecation warning:

```sh
npm WARN react-konva@16.7.0 requires a peer of react@16.6.3 but none is installed. You must install peer dependencies yourself.
npm WARN react-konva@16.7.0 requires a peer of react-dom@16.6.3 but none is installed. You must install peer dependencies yourself.
```

Can the `react` & `react-dom` peerDependencies be updated please?

Thanks for all your great work @lavrton! 🙂

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Closes #309

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
     "konva"
   ],
   "bugs": "https://github.com/konvajs/react-konva/issues",
-  "licenses": "MIT",
```
Comment (lavrton) [inline package.json]: Why did you remove the license?

Comment (MichaelDeBoey) [reply package.json]: @lavrton It's already under the `licence` key, this is a duplicate `licence`**`s`** (multiple) key

Comment (lavrton) [reply package.json]: Ah, right! I didn't notice that.

**Comment by lavrton** [comment]
THanks!

**Comment by MichaelDeBoey** [comment]
You're welcome @lavrton 🙂

I know it's Christmas eve, but don't forget to tag a new release please 😉
